code/datums/wires/wires.dm

```diff
@@ -303,3 +303,17 @@ var/const/POWER = 8
 	if(wires_status == (1 << wire_count) - 1)
 		return 1
 	return 0
+
+/datum/wires/proc/npc_tamper(mob/living/L)
+	if(!wires.len)
+		return
+
+	var/wire_to_screw = pick(wires)
+
+	if(IsColourCut(wire_to_screw) || prob(50)) //CutWireColour() proc handles both cutting and mending wires. If the wire is already cut, always mend it back. Otherwise, 50% to cut it and 50% to pulse it
+		CutWireColour(wire_to_screw, L)
```

Comment:
It has been a while since I've looked at wirecode, won't this expose gremlins to shocks? Which should be quite lethal at their health